swap/src/config.rs

```diff
@@ -0,0 +1,47 @@
+use anyhow::Result;
+use libp2p::multiaddr::Multiaddr;
+use serde::{Deserialize, Serialize};
+use std::{fs, path::Path};
+use url::Url;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Config {
+    pub bitcoind_url: Url,
+    pub listen_addr: Multiaddr,
```

Comment:
Perhaps this should be an option? If the node is being run as Bob we don't need this.

Reply:
Ah right.... It is actually the address to dial to connect to a peer, not the listen addr.